### ISSUE TITLE
Towny won't show up on the dynmap

### ISSUE THREAD

**Issue by matthewblack04** [TownyAdvanced/Towny#4222]
description below in comment

**Comment by matthewblack04** [comment]
The full issue I have is that towny will not show up on the dynmap whatsoever. I tried a lot of available versions of dynmap towny and as of now i'm using dynmap towny 0.76 (as per a suggestion) however versions like 0.81 have the same problems. 
My dynmap version is Dynmap 3.1-Beta 2 and my towny version is 0.96.2.0

The problem my console will give is this, and it will repeat about every 30 seconds:
27.07 12:07:35 [Server] WARN Plugin Dynmap-Towny v0.76 generated an exception while executing task 78
27.07 12:07:35 [Server] INFO java.lang.NoSuchMethodError: com.palmergames.bukkit.towny.TownyUniverse.getTownsMap()Ljava/util/concurrent/ConcurrentHashMap;
27.07 12:07:35 [Server] INFO at org.dynmap.towny.DynmapTownyPlugin.updateTownPlayerSets(DynmapTownyPlugin.java:345) ~[?:?]
27.07 12:07:35 [Server] INFO at org.dynmap.towny.DynmapTownyPlugin.access$100(DynmapTownyPlugin.java:51) ~[?:?]
27.07 12:07:35 [Server] INFO at org.dynmap.towny.DynmapTownyPlugin$TownyUpdate.run(DynmapTownyPlugin.java:300) ~[?:?]
27.07 12:07:35 [Server] INFO at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.15.2.jar:git-Paper-143]
27.07 12:07:35 [Server] INFO at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54) ~[patched_1.15.2.jar:git-Paper-143]
27.07 12:07:35 [Server] INFO at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.15.2.jar:git-Paper-143]
27.07 12:07:35 [Server] INFO at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_211]
27.07 12:07:35 [Server] INFO at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_211]
27.07 12:07:35 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_211]
27.07 12:07:50 [Server] WARN Plugin Dynmap-Towny v0.76 generated an exception while executing task 78
27.07 12:07:50 [Server] INFO java.lang.NoSuchMethodError: com.palmergames.bukkit.towny.TownyUniverse.getTownsMap()Ljava/util/concurrent/ConcurrentHashMap;
27.07 12:07:50 [Server] INFO at org.dynmap.towny.DynmapTownyPlugin.updateTownPlayerSets(DynmapTownyPlugin.java:345) ~[?:?]
27.07 12:07:50 [Server] INFO at org.dynmap.towny.DynmapTownyPlugin.access$100(DynmapTownyPlugin.java:51) ~[?:?]
27.07 12:07:50 [Server] INFO at org.dynmap.towny.DynmapTownyPlugin$TownyUpdate.run(DynmapTownyPlugin.java:300) ~[?:?]
27.07 12:07:50 [Server] INFO at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.15.2.jar:git-Paper-143]
27.07 12:07:50 [Server] INFO at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54) ~[patched_1.15.2.jar:git-Paper-143]
27.07 12:07:50 [Server] INFO at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.15.2.jar:git-Paper-143]
27.07 12:07:50 [Server] INFO at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_211]
27.07 12:07:50 [Server] INFO at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_211]
27.07 12:07:50 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_211]

I've tried a lot of things, followed about every suggestion and way to fix it, tried to change things in the configs of both dynmap and dynmap towny, but to this moment I can't pinpoint, nor fix the issue at hand.
